docs/contrib/quality.md

```diff
@@ -0,0 +1,38 @@
+# Quality checks
+
+There are a few checks that we apply to the configuration of this
+site. These checks run automatically via GitHub Actions workflows when
+you [send your PR](modifications.md):
+
+* We check the commit message with
+  [gitlint](https://jorisroovers.com/gitlint/), and enforce the
+  [Conventional
+  Commits](https://www.conventionalcommits.org/en/v1.0.0/) commit
+  message style.
+* We check whether the documentation still builds correctly, with your
+  change applied.
+* We check to make sure that no internal or external links in the
+  documentation are dead. This is one example where the checks might
+  fail through no fault of yours at all — some external link may have
```

Comment:
The "at all" is not needed, I would even say that it is even wrong. So, in my opinion, the sentence should start like this: "This is one example where the checks might fail through no fault of yours...".

Reply:
I agree.